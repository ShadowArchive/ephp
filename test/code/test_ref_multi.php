<?php

$a = 100;

$b = [&$a, &$a, &$a];
var_dump($b);

$a = 10;
var_dump($b);

unset($a);
var_dump($b);

unset($b[0]);
var_dump($b);

unset($b[1]);
var_dump($b);

unset($b[2]);
var_dump($b);
