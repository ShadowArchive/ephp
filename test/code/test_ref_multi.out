array(3) {
  [0]=>
  &int(100)
  [1]=>
  &int(100)
  [2]=>
  &int(100)
}
array(3) {
  [0]=>
  &int(10)
  [1]=>
  &int(10)
  [2]=>
  &int(10)
}
array(3) {
  [0]=>
  &int(10)
  [1]=>
  &int(10)
  [2]=>
  &int(10)
}
array(2) {
  [1]=>
  &int(10)
  [2]=>
  &int(10)
}
array(1) {
  [2]=>
  int(10)
}
array(0) {
}
